ui/src/pages/clearing/MTMCalculationModal.tsx

```diff
@@ -39,6 +39,12 @@ const MTMCalculationModal: React.FC<ServicePageProps<Service> & MTMProps> = ({
     }
   }, [member]);
 
+  useEffect(() => {
+    if (!currencyId && assets.length > 0) {
+      setCurrencyId(assets.find(c => c.payload.assetId.label === 'USD')?.payload.assetId);
+    }
+  }, [currencyId, assets]);
```

Comment:
Does this mean the UI currently only does MtM with USD, and no other asset/currency?

Reply:
@alexgraham-da can elaborate on this more than I can but my current understanding is that it works only with `USD` atm.

Reply:
yes, that is correct. at some point this will be expanded, but that was the minimum requirement for the baymarkets demo, so we left it at that for the time being.